sflow/agent.go

```diff
@@ -103,7 +110,16 @@ func (sfa *Agent) feedFlowTable() {
 				// records each generating Packets.
 				sfa.FlowTable.FeedWithSFlowSample(&sample, bpf)
 			}
+			var counters []layers.SFlowCounterSample
+			for _, sample := range sflowPacket.CounterSamples {
```

Comment:
this loop doesn't seem to be useful why not pushing sflowPacket.CounterSamples directly ? In the future we will need to define what we want to keep, not storing everything but only what make sense for us.

Reply:
Done